src/main/java/seedu/eduke8/parser/MenuParser.java

```diff
@@ -27,6 +26,16 @@
  */
 public class MenuParser implements Parser {
     private static final Logger LOGGER = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
+    private static final String TOPICS_INDICATOR = "t/";
+    private static final String QN_INDICATOR = "n/";
+    private static final String BOOKMARK_LIST = "listing";
+    private static final String COMMAND_ABOUT = "about";
+    private static final String COMMAND_HELP = "help";
+    private static final String COMMAND_TOPICS = "topics";
+    private static final String COMMAND_TEXTBOOK = "textbook";
+    private static final String COMMAND_QUIZ = "quiz";
+    private static final String COMMAND_BOOKMARK = "bookmark";
+    private static final String COMMAND_EXIT = "exit";
```

Comment:
Thank you for helping me replace my magic literals as well! All of it looks good to me. Will merge it in!!